### PR TITLE
check required permissions are displayed for files

### PR DESCRIPTION
**Pull request by HariBhandari07**
## Description

adds collaborator tests for file
## Related Issue

- Fixes https://github.com/owncloud/phoenix/issues/1768

## Motivation and Context

This tests adds collaborator test for file sharing
## How Has This Been Tested?

CI
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUILogin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5302/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5302/

<p>





</p></details>

File: tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature [hunk 1]
```diff
@@ -385,3 +385,53 @@ Feature: Sharing files and folders with internal users
...
+      | share_with  | user1               |
+      | file_target | /lorem (2).txt      |
+      | item_type   | file                |
+      | permissions | <permissions>       |
+    Examples:
+      | initial-permissions | permissions         |
+      | read, change        | read, share, change |
+      | read                | read, share         |
+
+  Scenario: Change permissions of the previously shared file
```
Comment (individual-it) [inline tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature]:
```suggestion
  Scenario: Delete all custom permissions of the previously shared file
```

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5413/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5413/

<p>















</p></details>

**Pull request merged**